editions/tw5.com/tiddlers/filters/substitute Operator.tid

```diff
@@ -20,8 +20,8 @@ The <<.op substitute>> operator replaces any placeholders in the input titles in
 |placeholder syntax|description|h
 |`$n$`|Text substitution of a parameter provided to the operator, where n is the position of the parameter starting with 1 for the first parameter. Unmatched placeholders pass through unchanged.|
 |`$(varname)$`|Text substitution of a variable. Undefined variables are replaced with an empty string.|
-|`${ filter expression }$`|Text substitution with the first result of evaluating the filter expression.  |
+|`${ filter expression }$`|Text substitution with the first result of evaluating the filter expression. So if the expression returns multiple titles, only the first one can be used. |
```

Comment:
I would suggest:
"If the filter expressions returns multiple titles only the first title is used".

Also note that the current wording is consistent with the tiddler "Filtered Attribute Values", see:
https://tiddlywiki.com/#Filtered%20Attribute%20Values